services/s3/pom.xml

```diff
@@ -129,6 +129,12 @@
             <version>${awsjavasdk.version}</version>
         </dependency>
         <!-- Test Dependencies -->
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>retries</artifactId>
+            <scope>test</scope>
+            <version>${awsjavasdk.version}</version>
+        </dependency>
```

Comment:
Is there any reason for this change?

Reply:
Configuring retry strategy on multipartClient in new test class S3MultipartClientGetObjectWiremockTest, to increase maxAttempts and set circuitBreakerEnabled(false)